jest.browser.json

```diff
@@ -3,7 +3,7 @@
   "coverageDirectory": "./build2/reports/coverage",
   "restoreMocks": true,
   "moduleNameMapper": {
-    "^OktaAuth(.*)$": "<rootDir>/jquery/$1"
+    "^OktaAuth$": "<rootDir>/lib/browser/browserIndex"
```

Comment:
I think this is okay, but am wondering if (for more complete test coverage) we should instead be reading from package.json (i.e. here we'd read from the browser property, and in the server config read from main). Would require changing the jest config from a json file to js.